Classes/Public/Manager/Objects/OCTCall.h

```diff
@@ -28,6 +28,12 @@
 @property OCTCallStatus status;
 
 /**
+ * The friend who started the call.
+ * Nil if the you started the call yourself.
+ **/
+@property OCTFriend *caller;
+
+/**
```

Comment:
You can also add `isOutgoing` method to OCTCall, which would just check for a caller

```
- (BOOL)isOutgoing
{
    return (self.caller == nil);
}
```